app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -61,11 +76,17 @@ class SubmitForm526 < Job
       #
       # rubocop:disable Metrics/MethodLength
       def perform(submission_id)
+        @submission_id = submission_id
+
         Raven.tags_context(source: '526EZ-all-claims')
         super(submission_id)
 
         submission.prepare_for_evss!
         with_tracking('Form526 Submission', submission.saved_claim_id, submission.id, submission.bdd?) do
+          # This instantiates the service as defined by the inheriting object
+          # TODO: this meaningless variable assignment is required for the specs to pass, which
+          # indicates a problematic coupling of implementation and test logic.  This should eventually
+          # be addressed to make this service and test more robust and readable.
```

Comment:
I appreciate the callout here. Way before we were complaining about data visibility and test environments, we had also brought up taking a second look at automated tests, in particular those involving VCR, due to a couple ineffective, potentially problematic implementations we discovered. That was explored for a bit, but not sure if it landed in anyone's notebook... maybe a note to bring it up again once the high priority stuff is finished.

I'm curious what tests were failing, and if it'd be too OOS to decouple (probably is).

Reply:
^ Ah and you addressed this preemptively with a comment that I missed

Reply:
> That was explored for a bit, but not sure if it's still ongoing... maybe a note to bring it up again once the high priority stuff is finished.

Also I extra lied. This is being actively looked at! Maybe this can be brought up to Cory and/or Steve

Reply:
Nice!  Yeah I didn't look too closely at the specs, but there were > %50 failures in the related spec file.